argus/action_manager/windows.py

```diff
@@ -313,15 +313,16 @@ def mkfile(self, path):
         """
         if self.is_file(path):
             LOG.warning("File '{}' already exists. LastWriteTime and"
-                        " LastAccessTime will be updated.").format(path)
+                        " LastAccessTime will be updated.".format(path))
             self._client.run_command_with_retry(
                 "echo $null >> '{}'".format(path),
                 command_type=util.POWERSHELL)
         elif self.is_dir(path):
             raise exceptions.ArgusCLIError(
                 "Path '{}' leads to a"
                 " directory.".format(path))
-        self._new_item(path, self._FILE)
+        else:
```

Comment:
This isn't a code typo, you could change the commit message to be more explicit.